projects/arlas-maplibre/src/lib/map/ArlasMaplibreGL.ts

```diff
@@ -222,7 +222,7 @@ export class ArlasMaplibreGL extends AbstractArlasMapGL {
   protected _initMapMoveEvents(): void {
     super._initMapMoveEvents();
     this.getMapProvider().on('mousedown', (e) =>
-      this._updateStartLngLat(e)
+      this._updatestartLngLat(e)
```

Comment:
Why has this line changed? There might have been a mishap when replacing startlngLat strings